gossip/src/cluster_info.rs

```diff
@@ -2432,15 +2445,23 @@ impl ClusterInfo {
         for (from_addr, packet) in packets {
             match packet {
                 Protocol::PullRequest(filter, caller) => {
-                    pull_requests.push((from_addr, filter, caller))
+                    if verify_gossip_addr(&caller) {
```

Comment:
where is this method defined? the one in this file has a signature with more params

Reply:
line 2427

Reply:
ah derp... totally missed that closure declaration